Source/Core/Core/CoreParameter.cpp

```diff
@@ -373,6 +373,14 @@ void SCoreStartupParameter::CheckMemcardPath(std::string& memcardPath, std::stri
 	}
 }
 
+IVolume::ELanguage SCoreStartupParameter::GetCurrentLanguage(bool wii)
+{
+	if (wii)
+		return (IVolume::ELanguage)SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.LNG");
```

Comment:
What should be done instead? Some sort of check like `if (language > LANGUAGE_UNKNOWN) language = LANGUAGE_UNKNOWN;`?

I don't think most users manually will set invalid values, and I don't think the GUI will do something weird with invalid languages either, but I suppose it's worth doing something about if it isn't too complicated.